specs/v1.yaml

```diff
@@ -1651,6 +1875,57 @@ components:
                   This is useful for nonprofits to indicate when they have received and processed the grant.
                   Expressed in ISO 8601 format.
                   A null value indicates that the grant has not been acknowledged.
+    RecurringGrantCaptureRequest:
+      description: |-
+        The request to create and submit a monthly recurring grant. This is useful to capture a recurring grant intent associated with Connect workflow session.
+        The request should specify the grant amount. The grant will re-occur on the same day of the month that the grant was submitted.
+        The recurring grant will continue indefinitely until the donor runs out of funds or until the donor cancels the recurring grant with their DAF provider.
+        This is the amount submitted for processing by the DAF.
+      required: true
+      content:
+        application/json:
+          schema:
+            type: object
+            required:
+              - workflowSessionId
+              - amount
+
+            properties:
+              workflowSessionId:
```

Comment:
we should require a frequency field (enum) where the only value available right now is MONTHLY